docs/send-data/opentelemetry-collector/remote-management/processing-rules/index.md

```diff
@@ -1,13 +1,8 @@
 ---
 slug: /send-data/opentelemetry-collector/remote-management/processing-rules
-title: Processing Rules for OpenTelemetry (Beta)
+title: Processing Rules for OpenTelemetry
 description: Use Sumo Logic processing rules for an OpenTelemetry agent with an OpenTelemetry remote management (OTRM) source template.
```

Comment:
I see "Processing Rules" and "processing rules" in this article. Change all instances to lowercase.